### ISSUE TITLE
apt finds packages with search but not with install

### ISSUE THREAD

**Issue by izzergh** [kdeldycke/meta-package-manager#881]
### What happened?

The goal was to install `snapd` using `apt` via `mpm`.

`mpm search snapd` revealed the only package manager I had installed that could install `snapd` was `apt`, so I did

`mpm install snapd`, which failed as no package manager could find that package.

Trying `mpm -v DEBUG --apt install snapd` reveals something weird:
```shell
debug: Try to install snapd with apt.
debug: ► /usr/bin/apt search ^snapd$ --names-only --quiet
debug:   Sorting...
debug:   Full Text Search...
debug:   snapd/focal-updates 2.57.5+20.04ubuntu0.1 amd64
debug:     Daemon and tooling that enable snap packages
warning:   WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
warning: No snapd package found on apt.
```

`mpm` claims `apt` can't find a `snapd`, yet it reveals exactly one result somewhere in between.

I tested to see if it was an issue with that specific package, I tried the same with `git` with the same results, and I tried with some random package already installed via `apt` with the same result again.

Apologies for any weird formatting in the following sections, this is a brand-new install and I haven't had time to install and mess with my preferred terminal emulator.

### List package managers

```shell-session
debug: mpm, version 5.10.2
debug: {'username': '-', 'guid': 'faef69a225ee44fbd8a43337ae965cb', 'hostname': '-', 'hostfqdn': '-', 'uname': {'system': 'Linux', 'node': '-', 'release': '5.15.0-56-generic', 'version': '#62~20.04.1-Ubuntu SMP Tue Nov 22 21:24:20 UTC 2022', 'machine': 'x86_64', 'processor': 'x86_64'}, 'linux_dist_name': '', 'linux_dist_version': '', 'cpu_count': 4, 'fs_encoding': 'utf-8', 'ulimit_soft': 1024, 'ulimit_hard': 1048576, 'cwd': '-', 'umask': '0o2', 'python': {'argv': '-', 'bin': '-', 'version': '3.8.10 (default, Nov 14 2022, 12:59:47) [GCC 9.4.0]', 'compiler': 'GCC 9.4.0', 'build_date': 'Nov 14 2022 12:59:47', 'version_info': [3, 8, 10, 'final', 0], 'features': {'openssl': 'OpenSSL 1.1.1f  31 Mar 2020', 'expat': 'expat_2.2.9', 'sqlite': '3.31.1', 'tkinter': '', 'zlib': '1.2.11', 'unicode_wide': True, 'readline': True, '64bit': True, 'ipv6': True, 'threading': True, 'urandom': True}}, 'time_utc': '2022-12-30 06:41:19.771320', 'time_utc_offset': -5.0, '_eco_version': '1.0.1'}
debug: No initial population of managers selected by user.
debug: apm CLI not found.
debug: CLI found at /usr/bin/apt
debug: ► /usr/bin/apt --version
debug:   apt 2.0.9 (amd64)
debug: Extracted version: '2.0.9'
debug: Parsed version: <TokenizedString 2.0.9 => (<Token:string='2',integer=2>, <Token:string='0',integer=0>, <Token:string='9',integer=9>)>
debug: CLI found at /usr/bin/apt
debug: ► /usr/bin/apt version apt
error:   WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
error: 
error:   E: Invalid operation version
debug: brew CLI not found.
debug: cargo CLI not found.
debug: brew CLI not found.
debug: choco CLI not found.
debug: composer CLI not found.
debug: dnf CLI not found.
debug: emerge CLI not found.
debug: CLI found at /usr/bin/flatpak
debug: ► /usr/bin/flatpak --version
debug:   Flatpak 1.12.7
debug: Extracted version: '1.12.7'
debug: Parsed version: <TokenizedString 1.12.7 => (<Token:string='1',integer=1>, <Token:string='12',integer=12>, <Token:string='7',integer=7>)>
debug: gem CLI not found.
debug: mas CLI not found.
debug: npm CLI not found.
debug: opkg CLI not found.
debug: pacaur CLI not found.
debug: pacman CLI not found.
debug: paru CLI not found.
debug: CLI found at /usr/bin/python3
debug: ► /usr/bin/python3 -m pip --no-color --version
debug:   pip 22.3.1 from /home/is/.local/lib/python3.8/site-packages/pip (python 3.8)
debug: Extracted version: '22.3.1'
debug: Parsed version: <TokenizedString 22.3.1 => (<Token:string='22',integer=22>, <Token:string='3',integer=3>, <Token:string='1',integer=1>)>
debug: pipx CLI not found.
debug: scoop CLI not found.
debug: snap CLI not found.
debug: steamcmd CLI not found.
debug: code CLI not found.
debug: yarn CLI not found.
debug: yay CLI not found.
debug: yum CLI not found.
debug: zypper CLI not found.
╭────────────┬────────────────────┬────────────────┬──────────────────────┬────────────┬──────────╮
│ Manager ID │ Name               │ Supported      │ CLI                  │ Executable │ Version  │
├────────────┼────────────────────┼────────────────┼──────────────────────┼────────────┼──────────┤
│ apm        │ Atom's apm         │ ✓ (deprecated) │ ✘ apm not found      │            │          │
│ apt        │ APT                │ ✓              │ ✓ /usr/bin/apt       │ ✓          │ ✓ 2.0.9  │
│ apt-mint   │ Linux Mint's apt   │ ✓              │ ✓ /usr/bin/apt       │ ✓          │ ✘        │
│ brew       │ Homebrew Formulae  │ ✓              │ ✘ brew not found     │            │          │
│ cargo      │ Rust's cargo       │ ✓              │ ✘ cargo not found    │            │          │
│ cask       │ Homebrew Cask      │ ✘ macOS only   │ ✘ brew not found     │            │          │
│ choco      │ Chocolatey         │ ✘ Windows only │ ✘ choco not found    │            │          │
│ composer   │ PHP's Composer     │ ✓              │ ✘ composer not found │            │          │
│ dnf        │ DNF                │ ✓              │ ✘ dnf not found      │            │          │
│ emerge     │ Emerge             │ ✓              │ ✘ emerge not found   │            │          │
│ flatpak    │ Flatpak            │ ✓              │ ✓ /usr/bin/flatpak   │ ✓          │ ✓ 1.12.7 │
│ gem        │ Ruby Gems          │ ✓              │ ✘ gem not found      │            │          │
│ mas        │ Mac AppStore       │ ✘ macOS only   │ ✘ mas not found      │            │          │
│ npm        │ Node's npm         │ ✓              │ ✘ npm not found      │            │          │
│ opkg       │ OPKG               │ ✓              │ ✘ opkg not found     │            │          │
│ pacaur     │ Pacaur             │ ✓              │ ✘ pacaur not found   │            │          │
│ pacman     │ Pacman             │ ✓              │ ✘ pacman not found   │            │          │
│ paru       │ Paru               │ ✓              │ ✘ paru not found     │            │          │
│ pip        │ Pip                │ ✓              │ ✓ /usr/bin/python3   │ ✓          │ ✓ 22.3.1 │
│ pipx       │ Pipx               │ ✓              │ ✘ pipx not found     │            │          │
│ scoop      │ Scoop              │ ✘ Windows only │ ✘ scoop not found    │            │          │
│ snap       │ Snap               │ ✓              │ ✘ snap not found     │            │          │
│ steamcmd   │ Valve Steam        │ ✓              │ ✘ steamcmd not found │            │          │
│ vscode     │ Visual Studio Code │ ✓              │ ✘ code not found     │            │          │
│ yarn       │ Node's yarn        │ ✓              │ ✘ yarn not found     │            │          │
│ yay        │ Yay                │ ✓              │ ✘ yay not found      │            │          │
│ yum        │ YUM                │ ✓              │ ✘ yum not found      │            │          │
│ zypper     │ Zypper             │ ✓              │ ✘ zypper not found   │            │          │
╰────────────┴────────────────────┴────────────────┴──────────────────────┴────────────┴──────────╯
```


### Meta Package Manager version

```shell-session
mpm, version 5.10.2
{'username': '-', 'guid': '94a545308a1fc09e2fff8b2e21d925e', 'hostname': '-', 'hostfqdn': '-', 'uname': {'system': 'Linux', 'node': '-', 'release': '5.15.0-56-generic', 'version': '#62~20.04.1-Ubuntu SMP Tue Nov 22 21:24:20 UTC 2022', 'machine': 'x86_64', 'processor': 'x86_64'}, 'linux_dist_name': '', 'linux_dist_version': '', 'cpu_count': 4, 'fs_encoding': 'utf-8', 'ulimit_soft': 1024, 'ulimit_hard': 1048576, 'cwd': '-', 'umask': '0o2', 'python': {'argv': '-', 'bin': '-', 'version': '3.8.10 (default, Nov 14 2022, 12:59:47) [GCC 9.4.0]', 'compiler': 'GCC 9.4.0', 'build_date': 'Nov 14 2022 12:59:47', 'version_info': [3, 8, 10, 'final', 0], 'features': {'openssl': 'OpenSSL 1.1.1f  31 Mar 2020', 'expat': 'expat_2.2.9', 'sqlite': '3.31.1', 'tkinter': '', 'zlib': '1.2.11', 'unicode_wide': True, 'readline': True, '64bit': True, 'ipv6': True, 'threading': True, 'urandom': True}}, 'time_utc': '2022-12-30 06:44:16.915856', 'time_utc_offset': -5.0, '_eco_version': '1.0.1'}
```


### Meta Package Manager configuration

```shell-session
╭────────────────────────────┬────────────────────────────────────────┬─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┬──────┬──────────────────┬─────────┬─────────────────────────┬─────────────────────────────────────────────────────┬─────────────────────────────────────────────────────┬─────────────╮
│ ID                         │ Class                                  │ Spec.                                                                                                                                                                                                                                                                                                                                                                                                                           │ Type │ Allowed in conf? │ Exposed │ Env. var.               │ Default                                             │ Value                                               │ Source      │
├────────────────────────────┼────────────────────────────────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┼──────┼──────────────────┼─────────┼─────────────────────────┼─────────────────────────────────────────────────────┼─────────────────────────────────────────────────────┼─────────────┤
│ mpm.all_managers           │ cloup._params.Option                   │ -a, --all-managers                                                                                                                                                                                                                                                                                                                                                                                                              │ bool │                  │ ✓       │ MPM_ALL_MANAGERS        │ False                                               │ False                                               │ DEFAULT     │
│ mpm.apm                    │ cloup._params.Option                   │ --apm                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_APM                 │ False                                               │ False                                               │ DEFAULT     │
│ mpm.apt                    │ cloup._params.Option                   │ --apt                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_APT                 │ False                                               │ False                                               │ DEFAULT     │
│ mpm.apt_mint               │ cloup._params.Option                   │ --apt-mint                                                                                                                                                                                                                                                                                                                                                                                                                      │ bool │                  │ ✘       │ MPM_APT_MINT            │ False                                               │ False                                               │ DEFAULT     │
│ mpm.bar_plugin_path        │ cloup._params.Option                   │ --bar-plugin-path                                                                                                                                                                                                                                                                                                                                                                                                               │ bool │                  │ ✘       │ MPM_BAR_PLUGIN_PATH     │ False                                               │ False                                               │ DEFAULT     │
│ mpm.brew                   │ cloup._params.Option                   │ --brew                                                                                                                                                                                                                                                                                                                                                                                                                          │ bool │                  │ ✘       │ MPM_BREW                │ False                                               │ False                                               │ DEFAULT     │
│ mpm.cargo                  │ cloup._params.Option                   │ --cargo                                                                                                                                                                                                                                                                                                                                                                                                                         │ bool │                  │ ✘       │ MPM_CARGO               │ False                                               │ False                                               │ DEFAULT     │
│ mpm.cask                   │ cloup._params.Option                   │ --cask                                                                                                                                                                                                                                                                                                                                                                                                                          │ bool │                  │ ✘       │ MPM_CASK                │ False                                               │ False                                               │ DEFAULT     │
│ mpm.choco                  │ cloup._params.Option                   │ --choco                                                                                                                                                                                                                                                                                                                                                                                                                         │ bool │                  │ ✘       │ MPM_CHOCO               │ False                                               │ False                                               │ DEFAULT     │
│ mpm.color                  │ click_extra.colorize.ColorOption       │ --color, --ansi / --no-color, --no-ansi                                                                                                                                                                                                                                                                                                                                                                                         │ bool │                  │ ✘       │ MPM_COLOR               │ True                                                │ True                                                │ DEFAULT     │
│ mpm.composer               │ cloup._params.Option                   │ --composer                                                                                                                                                                                                                                                                                                                                                                                                                      │ bool │                  │ ✘       │ MPM_COMPOSER            │ False                                               │ False                                               │ DEFAULT     │
│ mpm.config                 │ click_extra.config.ConfigOption        │ -C, --config CONFIG_PATH                                                                                                                                                                                                                                                                                                                                                                                                        │ str  │                  │ ✘       │ MPM_CONFIG              │ /home/is/.config/mpm/*.{toml,yaml,yml,json,ini,xml} │ /home/is/.config/mpm/*.{toml,yaml,yml,json,ini,xml} │ DEFAULT     │
│ mpm.description            │ cloup._params.Option                   │ --description                                                                                                                                                                                                                                                                                                                                                                                                                   │ bool │                  │ ✓       │ MPM_DESCRIPTION         │ False                                               │ False                                               │ DEFAULT     │
│ mpm.dnf                    │ cloup._params.Option                   │ --dnf                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_DNF                 │ False                                               │ False                                               │ DEFAULT     │
│ mpm.dry_run                │ cloup._params.Option                   │ -d, --dry-run                                                                                                                                                                                                                                                                                                                                                                                                                   │ bool │                  │ ✓       │ MPM_DRY_RUN             │ False                                               │ False                                               │ DEFAULT     │
│ mpm.emerge                 │ cloup._params.Option                   │ --emerge                                                                                                                                                                                                                                                                                                                                                                                                                        │ bool │                  │ ✘       │ MPM_EMERGE              │ False                                               │ False                                               │ DEFAULT     │
│ mpm.exclude                │ cloup._params.Option                   │ -e, --exclude [apm|apt|apt-mint|brew|cargo|cask|choco|composer|dnf|emerge|flatpak|gem|mas|npm|opkg|pacaur|pacman|paru|pip|pipx|scoop|snap|steamcmd|vscode|yarn|yay|yum|zypper]                                                                                                                                                                                                                                                  │ list │                  │ ✓       │ MPM_EXCLUDE             │                                                     │                                                     │ DEFAULT     │
│ mpm.flatpak                │ cloup._params.Option                   │ --flatpak                                                                                                                                                                                                                                                                                                                                                                                                                       │ bool │                  │ ✘       │ MPM_FLATPAK             │ False                                               │ False                                               │ DEFAULT     │
│ mpm.gem                    │ cloup._params.Option                   │ --gem                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_GEM                 │ False                                               │ False                                               │ DEFAULT     │
│ mpm.help                   │ click_extra.colorize.HelpOption        │ -h, --help                                                                                                                                                                                                                                                                                                                                                                                                                      │ bool │                  │ ✘       │ MPM_HELP                │ False                                               │ False                                               │ DEFAULT     │
│ mpm.ignore_auto_updates    │ cloup._params.Option                   │ --ignore-auto-updates / --include-auto-updates                                                                                                                                                                                                                                                                                                                                                                                  │ bool │                  │ ✓       │ MPM_IGNORE_AUTO_UPDATES │ True                                                │ True                                                │ DEFAULT     │
│ mpm.manager                │ cloup._params.Option                   │ -m, --manager [apm|apt|apt-mint|brew|cargo|cask|choco|composer|dnf|emerge|flatpak|gem|mas|npm|opkg|pacaur|pacman|paru|pip|pipx|scoop|snap|steamcmd|vscode|yarn|yay|yum|zypper]                                                                                                                                                                                                                                                  │ list │                  │ ✓       │ MPM_MANAGER             │                                                     │                                                     │ DEFAULT     │
│ mpm.mas                    │ cloup._params.Option                   │ --mas                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_MAS                 │ False                                               │ False                                               │ DEFAULT     │
│ mpm.npm                    │ cloup._params.Option                   │ --npm                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_NPM                 │ False                                               │ False                                               │ DEFAULT     │
│ mpm.opkg                   │ cloup._params.Option                   │ --opkg                                                                                                                                                                                                                                                                                                                                                                                                                          │ bool │                  │ ✘       │ MPM_OPKG                │ False                                               │ False                                               │ DEFAULT     │
│ mpm.output_format          │ click_extra.tabulate.TableFormatOption │ -o, --output-format [asciidoc|csv|csv-excel|csv-excel-tab|csv-unix|double_grid|double_outline|fancy_grid|fancy_outline|github|grid|heavy_grid|heavy_outline|html|jira|json|latex|latex_booktabs|latex_longtable|latex_raw|mediawiki|mixed_grid|mixed_outline|moinmoin|orgtbl|outline|pipe|plain|presto|pretty|psql|rounded_grid|rounded_outline|rst|simple|simple_grid|simple_outline|textile|tsv|unsafehtml|vertical|youtrack] │ str  │                  │ ✘       │ MPM_OUTPUT_FORMAT       │ rounded_outline                                     │ rounded_outline                                     │ DEFAULT     │
│ mpm.pacaur                 │ cloup._params.Option                   │ --pacaur                                                                                                                                                                                                                                                                                                                                                                                                                        │ bool │                  │ ✘       │ MPM_PACAUR              │ False                                               │ False                                               │ DEFAULT     │
│ mpm.pacman                 │ cloup._params.Option                   │ --pacman                                                                                                                                                                                                                                                                                                                                                                                                                        │ bool │                  │ ✘       │ MPM_PACMAN              │ False                                               │ False                                               │ DEFAULT     │
│ mpm.paru                   │ cloup._params.Option                   │ --paru                                                                                                                                                                                                                                                                                                                                                                                                                          │ bool │                  │ ✘       │ MPM_PARU                │ False                                               │ False                                               │ DEFAULT     │
│ mpm.pip                    │ cloup._params.Option                   │ --pip                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_PIP                 │ False                                               │ False                                               │ DEFAULT     │
│ mpm.pipx                   │ cloup._params.Option                   │ --pipx                                                                                                                                                                                                                                                                                                                                                                                                                          │ bool │                  │ ✘       │ MPM_PIPX                │ False                                               │ False                                               │ DEFAULT     │
│ mpm.scoop                  │ cloup._params.Option                   │ --scoop                                                                                                                                                                                                                                                                                                                                                                                                                         │ bool │                  │ ✘       │ MPM_SCOOP               │ False                                               │ False                                               │ DEFAULT     │
│ mpm.show_params            │ click_extra.config.ShowParamsOption    │ --show-params                                                                                                                                                                                                                                                                                                                                                                                                                   │ bool │                  │ ✘       │ MPM_SHOW_PARAMS         │ False                                               │ True                                                │ COMMANDLINE │
│ mpm.snap                   │ cloup._params.Option                   │ --snap                                                                                                                                                                                                                                                                                                                                                                                                                          │ bool │                  │ ✘       │ MPM_SNAP                │ False                                               │ False                                               │ DEFAULT     │
│ mpm.sort_by                │ cloup._params.Option                   │ -s, --sort-by [manager_id|manager_name|package_id|package_name|version]                                                                                                                                                                                                                                                                                                                                                         │ str  │                  │ ✓       │ MPM_SORT_BY             │ manager_id                                          │ manager_id                                          │ DEFAULT     │
│ mpm.stats                  │ cloup._params.Option                   │ --stats / --no-stats                                                                                                                                                                                                                                                                                                                                                                                                            │ bool │                  │ ✓       │ MPM_STATS               │ True                                                │ True                                                │ DEFAULT     │
│ mpm.steamcmd               │ cloup._params.Option                   │ --steamcmd                                                                                                                                                                                                                                                                                                                                                                                                                      │ bool │                  │ ✘       │ MPM_STEAMCMD            │ False                                               │ False                                               │ DEFAULT     │
│ mpm.stop_on_error          │ cloup._params.Option                   │ --stop-on-error / --continue-on-error                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✓       │ MPM_STOP_ON_ERROR       │ False                                               │ False                                               │ DEFAULT     │
│ mpm.time                   │ click_extra.commands.TimerOption       │ --time / --no-time                                                                                                                                                                                                                                                                                                                                                                                                              │ bool │                  │ ✘       │ MPM_TIME                │ False                                               │ False                                               │ DEFAULT     │
│ mpm.verbosity              │ click_extra.logging.VerbosityOption    │ -v, --verbosity LEVEL                                                                                                                                                                                                                                                                                                                                                                                                           │ str  │                  │ ✘       │ MPM_VERBOSITY           │ INFO                                                │ INFO                                                │ DEFAULT     │
│ mpm.version                │ click_extra.version.VersionOption      │ --version                                                                                                                                                                                                                                                                                                                                                                                                                       │ bool │                  │ ✘       │ MPM_VERSION             │ False                                               │ False                                               │ DEFAULT     │
│ mpm.vscode                 │ cloup._params.Option                   │ --vscode                                                                                                                                                                                                                                                                                                                                                                                                                        │ bool │                  │ ✘       │ MPM_VSCODE              │ False                                               │ False                                               │ DEFAULT     │
│ mpm.xkcd                   │ cloup._params.Option                   │ -x, --xkcd                                                                                                                                                                                                                                                                                                                                                                                                                      │ bool │                  │ ✓       │ MPM_XKCD                │ False                                               │ False                                               │ DEFAULT     │
│ mpm.yarn                   │ cloup._params.Option                   │ --yarn                                                                                                                                                                                                                                                                                                                                                                                                                          │ bool │                  │ ✘       │ MPM_YARN                │ False                                               │ False                                               │ DEFAULT     │
│ mpm.yay                    │ cloup._params.Option                   │ --yay                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_YAY                 │ False                                               │ False                                               │ DEFAULT     │
│ mpm.yum                    │ cloup._params.Option                   │ --yum                                                                                                                                                                                                                                                                                                                                                                                                                           │ bool │                  │ ✘       │ MPM_YUM                 │ False                                               │ False                                               │ DEFAULT     │
│ mpm.zypper                 │ cloup._params.Option                   │ --zypper                                                                                                                                                                                                                                                                                                                                                                                                                        │ bool │                  │ ✘       │ MPM_ZYPPER              │ False                                               │ False                                               │ DEFAULT     │
│ mpm.backup.merge           │ cloup._params.Option                   │ --merge                                                                                                                                                                                                                                                                                                                                                                                                                         │ bool │                  │ ✓       │ MPM_MERGE               │ False                                               │ False                                               │ DEFAULT     │
│ mpm.backup.overwrite       │ cloup._params.Option                   │ --overwrite, --force, --replace                                                                                                                                                                                                                                                                                                                                                                                                 │ bool │                  │ ✓       │ MPM_OVERWRITE           │ False                                               │ False                                               │ DEFAULT     │
│ mpm.backup.toml_path       │ cloup._params.Argument                 │ [TOML_PATH]                                                                                                                                                                                                                                                                                                                                                                                                                     │ str  │                  │ ✓       │                         │ -                                                   │ -                                                   │ DEFAULT     │
│ mpm.backup.update_version  │ cloup._params.Option                   │ --update-version                                                                                                                                                                                                                                                                                                                                                                                                                │ bool │                  │ ✓       │ MPM_UPDATE_VERSION      │ False                                               │ False                                               │ DEFAULT     │
│ mpm.install.packages_specs │ cloup._params.Argument                 │ PACKAGES_SPECS...                                                                                                                                                                                                                                                                                                                                                                                                               │ list │                  │ ✓       │                         │                                                     │                                                     │ DEFAULT     │
│ mpm.installed.duplicates   │ cloup._params.Option                   │ -d, --duplicates                                                                                                                                                                                                                                                                                                                                                                                                                │ bool │                  │ ✓       │ MPM_DUPLICATES          │ False                                               │ False                                               │ DEFAULT     │
│ mpm.outdated.plugin_output │ cloup._params.Option                   │ --plugin-output                                                                                                                                                                                                                                                                                                                                                                                                                 │ bool │                  │ ✓       │ MPM_PLUGIN_OUTPUT       │ False                                               │ False                                               │ DEFAULT     │
│ mpm.remove.packages_specs  │ cloup._params.Argument                 │ PACKAGES_SPECS...                                                                                                                                                                                                                                                                                                                                                                                                               │ list │                  │ ✓       │                         │                                                     │                                                     │ DEFAULT     │
│ mpm.restore.toml_files     │ cloup._params.Argument                 │ TOML_FILES...                                                                                                                                                                                                                                                                                                                                                                                                                   │ list │                  │ ✓       │                         │                                                     │                                                     │ DEFAULT     │
│ mpm.search.exact           │ cloup._params.Option                   │ --exact / --fuzzy                                                                                                                                                                                                                                                                                                                                                                                                               │ bool │                  │ ✓       │ MPM_EXACT               │ False                                               │ False                                               │ DEFAULT     │
│ mpm.search.extended        │ cloup._params.Option                   │ --extended / --id-name-only                                                                                                                                                                                                                                                                                                                                                                                                     │ bool │                  │ ✓       │ MPM_EXTENDED            │ False                                               │ False                                               │ DEFAULT     │
│ mpm.search.query           │ cloup._params.Argument                 │ QUERY                                                                                                                                                                                                                                                                                                                                                                                                                           │ str  │                  │ ✓       │                         │                                                     │                                                     │ DEFAULT     │
│ mpm.search.refilter        │ cloup._params.Option                   │ --refilter / --no-refilter                                                                                                                                                                                                                                                                                                                                                                                                      │ bool │                  │ ✓       │ MPM_REFILTER            │ True                                                │ True                                                │ DEFAULT     │
│ mpm.upgrade.all            │ cloup._params.Option                   │ -A, --all                                                                                                                                                                                                                                                                                                                                                                                                                       │ bool │                  │ ✓       │ MPM_ALL                 │ False                                               │ False                                               │ DEFAULT     │
│ mpm.upgrade.packages_specs │ cloup._params.Argument                 │ [PACKAGES_SPECS]...                                                                                                                                                                                                                                                                                                                                                                                                             │ list │                  │ ✓       │                         │                                                     │                                                     │ DEFAULT     │
╰────────────────────────────┴────────────────────────────────────────┴─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┴──────┴──────────────────┴─────────┴─────────────────────────┴─────────────────────────────────────────────────────┴─────────────────────────────────────────────────────┴─────────────╯
```

**Comment by kdeldycke** [comment]
There was an issue in the regex parsing `apt`'s results. I just fixed it upstream in 8bd5a770a5171b5d78fac7b771b64ce2c7dc4145.

The fix will be available in the upcoming release of `mpm` 5.13.0.

**Comment by kdeldycke** [comment]
FYI meta-package-manager v5.13.0 has just been released.
